website/content/en/preview/reference/metrics.md

```diff
@@ -10,254 +10,327 @@ description: >
 Karpenter makes several metrics available in Prometheus format to allow monitoring cluster provisioning status. These metrics are available by default at `karpenter.karpenter.svc.cluster.local:8000/metrics` configurable via the `METRICS_PORT` environment variable documented [here](../settings)
 ### `karpenter_build_info`
 A metric with a constant '1' value labeled by version from which karpenter was built.
+- Stability Level: STABLE
 
 ## Nodepool Metrics
 
 ### `karpenter_nodepool_usage`
 The amount of resources that have been provisioned for a nodepool. Labeled by nodepool name and resource type.
+- Stability Level: STABLE
 
 ### `karpenter_nodepool_limit`
 Limits specified on the nodepool that restrict the quantity of resources provisioned. Labeled by nodepool name and resource type.
+- Stability Level: STABLE
 
 ### `karpenter_nodepool_allowed_disruptions`
 The number of nodes for a given NodePool that can be concurrently disrupting at a point in time. Labeled by NodePool. Note that allowed disruptions can change very rapidly, as new nodes may be created and others may be deleted at any point.
+- Stability Level: STABLE
 
 ## Nodeclaims Metrics
 
 ### `karpenter_nodeclaims_termination_duration_seconds`
 Duration of NodeClaim termination in seconds.
+- Stability Level: BETA
 
 ### `karpenter_nodeclaims_terminated_total`
 Number of nodeclaims terminated in total by Karpenter. Labeled by reason the nodeclaim was terminated and the owning nodepool.
+- Stability Level: STABLE
 
 ### `karpenter_nodeclaims_registered_total`
```

Comment:
This is fine for now. This is going in with the assumption that we will be dropping these metrics in the upstream code